wasm/schema.go

```diff
@@ -0,0 +1,65 @@
+// Copyright © 2024 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build wasm
+
+package wasm
+
+import (
+	"fmt"
+
+	conduitv1 "github.com/conduitio/conduit-processor-sdk/proto/conduit/v1"
+	"google.golang.org/protobuf/proto"
+)
+
+func CreateSchema(req *conduitv1.CreateSchemaRequest) (*conduitv1.CreateSchemaResponse, error) {
+	buf, err := proto.Marshal(req)
```

Comment:
I think we could reuse the same global `buffer` 🤔 Or maybe have a global pool of buffers that we reuse. That would maybe be the safer option.